[2025-05-19T18:33:07.122+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-19T18:33:07.170+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: car_prices_etl.load_to_greenplum manual__2025-05-19T18:32:57.922313+00:00 [queued]>
[2025-05-19T18:33:07.194+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: car_prices_etl.load_to_greenplum manual__2025-05-19T18:32:57.922313+00:00 [queued]>
[2025-05-19T18:33:07.196+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-05-19T18:33:07.237+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_to_greenplum> on 2025-05-19 18:32:57.922313+00:00
[2025-05-19T18:33:07.254+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1433) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-19T18:33:07.258+0000] {standard_task_runner.py:72} INFO - Started process 1435 to run task
[2025-05-19T18:33:07.257+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'car_prices_etl', 'load_to_greenplum', 'manual__2025-05-19T18:32:57.922313+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/test_car_2.py', '--cfg-path', '/tmp/tmpwzglp7e4']
[2025-05-19T18:33:07.261+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask load_to_greenplum
[2025-05-19T18:33:07.369+0000] {task_command.py:467} INFO - Running <TaskInstance: car_prices_etl.load_to_greenplum manual__2025-05-19T18:32:57.922313+00:00 [running]> on host db4eb1a3da04
[2025-05-19T18:33:07.816+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='car_prices_etl' AIRFLOW_CTX_TASK_ID='load_to_greenplum' AIRFLOW_CTX_EXECUTION_DATE='2025-05-19T18:32:57.922313+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-19T18:32:57.922313+00:00'
[2025-05-19T18:33:07.819+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-19T18:33:07.884+0000] {test_car_2.py:132} INFO - >>> [START] load_to_greenplum
[2025-05-19T18:33:08.017+0000] {base.py:84} INFO - Retrieving connection 'postgres_default'
[2025-05-19T18:33:08.073+0000] {test_car_2.py:200} INFO - Successfully loaded 10/10 records to GreenPlum
[2025-05-19T18:33:08.075+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-19T18:33:08.112+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-19T18:33:08.113+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=car_prices_etl, task_id=load_to_greenplum, run_id=manual__2025-05-19T18:32:57.922313+00:00, execution_date=20250519T183257, start_date=20250519T183307, end_date=20250519T183308
[2025-05-19T18:33:08.225+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-19T18:33:08.279+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-19T18:33:08.285+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
